configure/CONFIG_SITE.Common.rhel9-x86_64

```diff
@@ -1,3 +1,9 @@
 
 # Need libtirpc on modern Linux distros
 TIRPC=YES
+
+TIRPC_PACKAGE_NAME=tirpc
+TIRPC_VERSION=1.3.5
+TIRPC_TOP=$(PACKAGE_SITE_TOP)/$(TIRPC_PACKAGE_NAME)/$(TIRPC_VERSION)
+TIRPC_LIB=$(TIRPC_TOP)/$(PKG_ARCH)/lib
+TIRPC_INCLUDE=$(TIRPC_TOP)/$(PKG_ARCH)/include/tirpc
```

Comment:
Would it be possible to put these blurbs in CONFIG_SITE? Even if these don't point to valid directories for some targets (or are not needed), it shouldn't matter unless the tirpc_DIR stuff is expanded.

Reply:
Yes, this makes sense. I agree.

Reply:
@JJL772, it's done. I tested and even the RTEMS ones built properly. Can you have another look, please?